lms/static/js/verify_student/views/image_input_view.js

```diff
@@ -0,0 +1,114 @@
+            var renderedHtml = _.template( $( this.template ).html(), {} );
+            $( this.el ).html( renderedHtml );
+
+            // Set the submit button to disabled by default
+            this.setSubmitButtonEnabled( false );
+
+            this.$input = $( 'input.image-upload' );
+            this.$preview = $( 'img.preview' );
+            this.$input.on('change', _.bind( this.handleInputChange, this ) );
+
+            // Initially hide the preview
+            this.displayImage( false );
+
+            return this;
+        },
+
+        handleInputChange: function( event ) {
+            var files = event.target.files,
+                reader = new FileReader();
+            if ( files[0] && files[0].type.match( 'image.[png|jpg|jpeg]' ) ) {
+                reader.onload = _.bind( this.handleImageUpload, this );
+                reader.onerror = _.bind( this.handleUploadError, this );
+                reader.readAsDataURL( files[0] );
+            }
+            else if ( files.length === 0 ) {
+                this.handleUploadError( false );
+            }
+            else {
+                this.handleUploadError( true );
+            }
+        },
+
+        handleImageUpload: function( event ) {
+            var imageData = event.target.result;
+            this.model.set( this.modelAttribute, imageData );
+            this.displayImage( imageData );
+            this.setSubmitButtonEnabled( true );
+
+            // Hide any errors we may have displayed previously
+            if ( this.errorModel ) {
+                this.errorModel.set({ shown: false });
+            }
+
+            this.trigger( 'imageCaptured' );
+        },
+
+        displayImage: function( imageData ) {
+            if ( imageData ) {
+                this.$preview
+                    .attr( 'src', imageData )
+                    .removeClass('is-hidden')
+                    .attr('aria-hidden', 'false');
+            } else {
+                this.$preview
+                    .attr( 'src', '' )
+                    .addClass('is-hidden')
+                    .attr('aria-hidden', 'true');
+            }
+        },
+
+        handleUploadError: function( displayError ) {
+            this.displayImage( null );
+            this.setSubmitButtonEnabled( false );
+            if ( this.errorModel ) {
+                if ( displayError ) {
+                    this.errorModel.set({
+                        errorTitle: gettext( 'Image Upload Error' ),
+                        errorMsg: gettext( 'Please verify that you have uploaded a valid image (PNG and JPEG).' ),
+                        shown: true
+                    });
+                } else {
+                    this.errorModel.set({
+                        shown: false
+                    });
+                }
+            }
+            this.trigger( 'error' );
+        },
+
+        setSubmitButtonEnabled: function( isEnabled ) {
+            $( this.$submitButton )
```

Comment:
See comment re. init function - should just be calling this.$submitButton without wrapping in jQuery DOM selection function